addons/website_crm/tests/test_website_visitor.py

```diff
@@ -65,3 +70,55 @@ def test_compute_email_phone(self):
         customer.write({'mobile': False})
         self.assertEqual(visitor.email, customer.email_normalized)
         self.assertEqual(visitor.mobile, lead_2.mobile)
+
+
+@tagged('website_visitor')
+class WebsiteVisitorTestsCrm(WebsiteVisitorTests):
```

Comment:
Why a new class by the way ? Could be inlined within current class that has only 1 test and inherit from both test classes (crm and visitor). That way everything is in everything.